changelogs/fragments/89-dynamic-user-password-in-client-my-cnf-and-correct-package-name.yml

```diff
@@ -1,3 +0,0 @@
-bugfixes:
```

Comment:
This changelog fragment must be kept. It is used to generate the changelog for the next community.proxysql release.

Reply:
I re-added the change log file 🥲